app/lib/frontend/models.dart

```diff
@@ -197,6 +201,10 @@ class PackageVersion extends db.ExpandoModel {
   @db.StringProperty(required: false)
   String uploader;
 
+  /// The publisher id at the time of the upload.
+  @db.StringProperty()
+  String publisherId;
```

Comment:
Do we have a use-case for this? If not I suggest we avoid tracking it..

We still plan to store the email of the uploader, right?

Reply:
I think it is important to track if a given PackageVersion was published while under the ownership of a given publisher.

Reply:
But we already track who uploaded it, so what does it matter whether they were given permission to do so through a publisher, or because they were uploader at the time... ?

I think we're best off leaving audit logging as a separate thing..

Reply:
But if the upload happened while under a publisher, we will hide the uploaderEmail, and display only the publisher. Assume we have 1.0.0 under user@example.com, 1.1.0 with user@secret.com under flutter.io, 2.0.0-dev1 under user@example.com (de-transitioned from publisher). When displaying 1.1.0 we'll need to know whether to display the userEmail or not.